Figures/Figures.cs

```diff
@@ -1,24 +1,50 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Figures
 {
     public abstract class Figure: IFigure
     {
+#region properties
```

Comment:
Using regions is a bad style. In most of the cases, if you feel that you need a region, this only means that your class is too big and should be split to simpler building blocks,